blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
Hmm, I think regardless of whether it's persisted or not, it makes more sense to have separate `isSelected` and `isRead` properties, although it's definitely worth checking whether or not the read status should be persisted.